### PR TITLE
fix: make compact tokenizer inner width: auto and make cozy mode horizontally scrollable

### PR DESCRIPTION
**Pull request by mikerodonnell89**
## Related Issue
Closes SAP/fundamental-styles# #774

### CODE REVIEW

**Comment by JKMarkowski** [comment]
Hello @mikerodonnell89 is it done by purpose?
Opera:
image

Ie11:
image

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-03-13 at 8 46 16 AM
- I think the width of the tokens is wrong. 
- The focus outline is cut at the right side 
- The naming of the PR title is not following the guidelines
- Would you please add Assignee, Projects and Milestones?

File: src/tokenizer.scss [hunk 1]
```diff
@@ -10,18 +10,24 @@ $block: #{$fd-namespace}-tokenizer;
...
   display: flex;
   align-items: center;
 
   &__inner {
+    &::-webkit-scrollbar {
+      display: none; // hide scrollbar but keep element scrollable for Chrome/Safari/Opera
+    }
+
+    padding: 0 $fd-tokenizer-spacing;
+    overflow: hidden;
```
Comment (mikerodonnell89) [inline src/tokenizer.scss]: The interaction spec states that the cozy (phone and tablet) tokenizer should be scrollable by swiping the tokens left and right.  So I think here we do need overflow-x: scroll

File: docs/pages/components/token.md [hunk 2]
```diff
@@ -280,3 +285,75 @@ To show focus on whole element, it's needed to add `.is-focus` into container.
 </div>
 {% endcapture %}
 {% include display-component.html component=compact-tokenizer %}
+
+## Tokenizer - Scrollable
+The Compact tokenizer is for large screens.
```
Comment (mikerodonnell89) [inline docs/pages/components/token.md]: This is the cozy tokenizer.  And the user won't have tab keys in cozy mode (phone/tablet).  In compact (desktop) mode they will have the "____more" text which when clicked opens the popover list containing the hidden tokens.

Comment (JKMarkowski) [reply docs/pages/components/token.md]: In my my approach, there is added `--scrollable`, which can be added in cozy and compact mode.

**Comment by JKMarkowski** [comment]
@mikerodonnell89 
This is for desktop
tokenizerdesktop

This one is on mobile/tablets/touch devices
tokenizer

File: src/tokenizer.scss [hunk 3]
```diff
@@ -38,7 +44,7 @@ $block: #{$fd-namespace}-tokenizer;
   &__input {
     border: none !important;
     min-width: 3rem !important;
-    padding: 0 $fd-tokenizer-spacing 0 0 !important;
+    padding: 0 !important;
```
Comment (jbadan) [inline src/tokenizer.scss]: Did we want to keep with the `important!` pattern here or switch to the new pattern we did with shellbar buttons?

Comment (JKMarkowski) [reply src/tokenizer.scss]: Now it's handled like on shellbar.

**Pull request merged**